integration/test_resources/vuln_scan/dirty.Dockerfile

```diff
@@ -0,0 +1 @@
+FROM node:latest
```

Comment:
Pin this version instead of using latest

Reply:
Wait, the latest Node container is dirty?!!! 😮

Reply:
Yup, quick and easy way to get a vulnerable container. Just add NPM.